### DISCUSSION TITLE
Request: Ability to increase gap between labels and content below

### DISCUSSION THREAD

**Discussion by pqpxo** [kinghat/tabbed-card#36] [Feature Requests]
Hello there,

More of a request than an issue ... Are you able to include a property/option to set the size of a gap between the bottom of the labels and the content shown below?

Many thanks and good work!

**Comment by kinghat** [comment]
could you mock up a graphic to show your use case? im just reusing the same web component design system that home assistant already uses so it looks the part. this seems pretty specific and i dont think i will be implementing everyones one off tweak requests. that said, im open minded. if you can show me a compelling reason for it :man_shrugging:

**Reply by pqpxo** [reply]
![tabbed](https://user-images.githubusercontent.com/43624115/201931469-30dd3b90-0e9b-4885-90a6-0c0ae5c5eb21.png)

Attached a screenshot demonstrating the gap I've mentioned. I've added a 'padding-top: 20px' in order to create the gap/spacing (see HTML/CSS on right of screenshot). Unsure whether this is the best or ideal place to add the padding.
